### ISSUE TITLE
Tear down & recreate testpilot.dev.mozaws.net to add tags for Reaper

### ISSUE THREAD

**Issue by lmorchard** [mozilla/testpilot#1129]
tl;dr: There's a chance our dev instance could be broken occasionally by an ops cleanup robot unless we add certain metadata to EC2. But, that requires tearing everything down and rebuilding it.

There's [a new ops service called "Reaper"](https://mana.mozilla.org/wiki/display/SVCOPS/Living+with+the+Reaper) that will destroy EC2 instances to clean up old VMs. It will skip destroying EC2 instances that have been tagged with `REAPER_SPARE_ME=true` and will notify an email address in the `Owner` tag.

Test Pilot dev uses Elastic Beanstalk, but Elastic Beanstalk manages EC2 instances for us. We can manually add the `REAPER_SPARE_ME` and `Owner` tags to those EC2 instances, but Elastic Beanstalk can destroy that EC2 instance during deploys & environment upgrades.

Elastic Beanstalk can be configured to automatically add those tags - but this can only be configured at initial creation time for an environment. They cannot be edited later. So, to get our dev server auto-tagging EC2 instances, we have to destroy it and recreate it by hand.

**Comment by lmorchard** [comment]
An update as I was filing this:

> I looked at your Elastic Beanstalk deployment and it seems like your test instance, 
> testpilot-dev, is in a Cloudformation, which is enough to keep it safe. We won't stop/terminate 
> anything in a Cloudformation. That said, getting those tags (Owner / REAPER_SPARE_ME) 
> passed down from your Cloudformation to EC2 is a good idea.
> 
> Cheers,
> Miles

So, it sounds like this is a good idea, but not super-urgent. It will also be somewhat disruptive as it will leave dev unavailable while rebuilding it and probably make CircleCI tests fail as they try to auto-deploy from master. 

It will also take a few hours, because this stuff always seems to take a few hours just from experience and random hiccups 😢

**Comment by ghost** [comment]
Don't fear the reaper! 🐮 🐄 🔔 🛎
